include/sound/sof.h

```diff
@@ -124,6 +124,10 @@ struct sof_dev_desc {
 	/* defaults for no codec mode */
 	const char *nocodec_tplg_filename;
 
+	/* information on supported IPCs */
+	int ipc_supported_mask;
+	int ipc_default;
```

Comment:
ipc_default is a enum so uint is better. And also ipc_supported_mask

Reply:
or define it as "enum sof_ipc_type ipc_default" so user can get more info easy.

Reply:
done